cluster/cluster.go

```diff
@@ -84,6 +84,9 @@ const (
 	WorkerThreads = util.WorkerThreads
 
 	serviceAccountTokenFileParam = "service-account-key-file"
+
+	unschedulableEtcdTaintKey    = "node-role.kubernetes.io/etcd=:NoExecute"
+	unschedulableControlTaintKey = "node-role.kubernetes.io/controlplane=:NoSchedule"
```

Comment:
Can you show reference for these reserved taints

Reply:
I put the similar taints together. I think we will understand better in this way.